### PR TITLE
doc: update README to align with the new brand name

### LINKED ISSUE #715: Update the README for the repository
### Is your feature request related to a problem?

I propose to update the README.md file for the repository with the following information:
- link to the overview of the organization
- purpose of the repository
- is it in active development or not, is it archived or not, any reasons for that
- links to relevant documentation (security audits, web site, design documents, etc.)

### What solution do you propose?

N/A

### What alternatives have you considered?

N/A

### Any additional context?

_No response_

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Resolve #715

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,35 +4,39 @@
...
+Notation is a CLI project to add signatures as standard items in the OCI registry ecosystem, and to build a set of simple tooling for signing and verifying these signatures. This should be viewed as similar security to checking git commit signatures, although the signatures are generic and can be used for additional purposes. Notation is an implementation of the [Notary Project specifications][notaryproject-specs].
 
-> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
+> **Note** The documentation for installing and using `notation` is available [here](https://notaryproject.dev/docs/). You can also find the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) to learn about the overall Notary Project.
 
 ## Table of Contents
 
   - [Documents](#documents)
```
Comment (yizha1) [inline README.md:13]: Replace it with `[Quick Start](#quick-start)`

File: README.md [hunk 2]
```diff
@@ -4,35 +4,39 @@
...
+Notation is a CLI project to add signatures as standard items in the OCI registry ecosystem, and to build a set of simple tooling for signing and verifying these signatures. This should be viewed as similar security to checking git commit signatures, although the signatures are generic and can be used for additional purposes. Notation is an implementation of the [Notary Project specifications][notaryproject-specs].
 
-> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
+> **Note** The documentation for installing and using `notation` is available [here](https://notaryproject.dev/docs/). You can also find the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) to learn about the overall Notary Project.
```
Comment (yizha1) [inline README.md:9]: Suggest moving `You can find the Notary Project readme` to line 8, not in the note with `installation` note

**Review by yizha1 — COMMENTED** [review]
Thanks @FeynmanZhou left a few comments

File: README.md (hunk 2 above)
Comment (Two-Hearts) [inline README.md:9]: Installing guide: `https://notaryproject.dev/docs/installation/` should be mentioned explicitly in the doc and should be as clear as possible. I think it is THE most important information of this README.

**Review by yizha1 — DISMISSED** [review]
Thanks @FeynmanZhou LGTM

**Review by Two-Hearts — APPROVED** [review]
There's still comment to resolve, otherwise LGTM.

File: README.md [hunk 3]
```diff
@@ -4,35 +4,41 @@
...
+Notation is a CLI project to add signatures as standard items in the OCI registry ecosystem, and to build a set of simple tooling for signing and verifying these signatures. This should be viewed as similar security to checking git commit signatures, although the signatures are generic and can be used for additional purposes. Notation is an implementation of the [Notary Project specifications][notaryproject-specs].
 
-> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
+You can find the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) to learn about the overall Notary Project.
+
+> **Note** The documentation for installing Notation CLI is available [here](https://notaryproject.dev/docs/installation/cli/).
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply README.md]: Resolved

**Review by Two-Hearts — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -4,35 +4,42 @@
...
   - [Release Management](#release-management)
   - [Support](#support)
   - [Code of Conduct](#code-of-conduct)
   - [License](#license)
 
-## Documents
+## Quick Start
 
 - [Quick start: Sign and validate a container image](https://notaryproject.dev/docs/quickstart/)
+- [Try Notation on an online playground](https://killercoda.com/notaryproject/scenario/notation)
```
Comment (gokarnm) [inline README.md:28]:
```suggestion
- [Try out Notation in this Killercoda interactive sandbox environment](https://killercoda.com/notaryproject/scenario/notation)
```

**Review by gokarnm — APPROVED** [review]
LGTM! Added a minor edit.

**Pull request merged**